### PR TITLE
ADM-358

### PR DESCRIPTION
**Pull request by fengxinhou**
## Summary
basic information in config step

## Before

null

## After

Project Name

Collection Date which will impact the state of verify and Next, see video

Required Data

### CODE REVIEW

File: frontend/__tests__/src/components/metrics/ConfigStep/DateRangePicker.test.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  })
+
+  it('should show error when input a invalid start date given init start date is null ', () => {
+    const { getByRole, getByText } = render(<DateRangePicker />)
+    const startDateInput = getByRole('textbox', { name: 'From' }) as HTMLInputElement
+
+    fireEvent.change(startDateInput, { target: { value: '02/03/' } })
+
+    expect(startDateInput.value).toEqual('02/03/')
+    expect(getByText('From')).toHaveStyle(`color: #d32f2f`)
```
Comment (guzhongren) [inline frontend/__tests__/src/components/metrics/ConfigStep/DateRangePicker.test.tsx:42]: Duplicated code for this check

File: frontend/__tests__/src/components/metrics/ConfigStep/DateRangePicker.test.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    fireEvent.change(endDateInput, { target: { value: today } })
+
+    expect(endDateInput.value).toEqual(today)
+  })
+
+  it('should show error when input a invalid start date given init start date is null ', () => {
+    const { getByRole, getByText } = render(<DateRangePicker />)
+    const startDateInput = getByRole('textbox', { name: 'From' }) as HTMLInputElement
+
+    fireEvent.change(startDateInput, { target: { value: '02/03/' } })
```
Comment (guzhongren) [inline frontend/__tests__/src/components/metrics/ConfigStep/DateRangePicker.test.tsx:39]: Invalid date `02/03` can be extracted into a fixture

File: frontend/src/components/metrics/ConfigStep/index.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      />
+
+      <h3>Collection Date</h3>
+      <RadioGroup defaultValue={REGULAR_CALENDAR}>
+        <FormControlLabel value={REGULAR_CALENDAR} control={<Radio />} label={REGULAR_CALENDAR} />
+        <FormControlLabel value={CHINA_CALENDAR} control={<Radio />} label={CHINA_CALENDAR} />
+      </RadioGroup>
+      <DateRangePicker />
+      <RequireDataSelections variant='standard' required error={isEmptyRequireData}>
+        <InputLabel id='demo-multiple-checkbox-label'>Require Data</InputLabel>
```
Comment (guzhongren) [inline frontend/src/components/metrics/ConfigStep/index.tsx:73]: Label should be `Required Data`

File: frontend/package.json [hunk 4]
```diff
@@ -24,11 +24,14 @@
     "**/*": "npx prettier --write . --ignore-unknown"
   },
   "dependencies": {
+    "@date-io/dayjs": "^2.16.0",
```
Comment (guzhongren) [inline frontend/package.json:27]: No usage for this package?

**Review by guzhongren — CHANGES_REQUESTED** [review]
Need some change for this PR

File: frontend/package.json (hunk 4 above)
Comment (fengxinhou) [reply frontend/package.json:27]: yes, DatePicker select value is Dayjs type.

**Review by LanxinWang — APPROVED** [review]

**Review by guzhongren — APPROVED** [review]
LGTM

**Review by LanxinWang — APPROVED** [review]

**Pull request merged**